docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
<p>It would look nicer if the repeated subsequences were highlighted, or more visible somehow.</p><p> </p><p>Try this (or something similar):</p><p> </p><pre class="ql-syntax" spellcheck="false">from matplotlib.patches import Rectangle
</pre><p> </p><pre class="ql-syntax" spellcheck="false">rect = Rectangle((0, 0), 20, 50, facecolor='lightgrey')
axs[0].add_patch(rect)
axs[0].plot(np.arange(20), t_1[:20], color='aquamarine', linewidth=10, alpha=0.5)

rect = Rectangle((5, 0), 20, 50, facecolor='lightgrey')
axs[1].add_patch(rect)
axs[1].plot(np.arange(5, 25), t_2[5:25], color='aquamarine', linewidth=10, alpha=0.5)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>